pkg/api/option.go

```diff
@@ -130,12 +129,7 @@ func WithProxy(proxyURL string) (Option, error) {
 
 // WithSSLCertFile overrides the default CA certs file to trust specified cert file.
 func WithSSLCertFile(filepath string) (Option, error) {
-	expanded, err := homedir.Expand(filepath)
-	if err != nil {
-		return nil, fmt.Errorf("failed expanding filepath %q: %s", filepath, err)
-	}
-
-	caCert, err := ioutil.ReadFile(expanded)
+	caCert, err := ioutil.ReadFile(filepath)
```

Comment:
:+1:

Reply:
We should also expand it for `--config` and `--log-file`.

Reply:
> We should also expand it for `--config` and `--log-file`.

Them are already being expanded.